platform/haiku/detect.py

```diff
@@ -24,7 +24,7 @@ def get_opts():
 def get_flags():
 	return [
 		('builtin_zlib', 'no'),
-		#('glew', 'yes'), # TODO: investigate the GLEW situation on Haiku
+		('glew', 'yes'), # TODO: investigate the GLEW situation on Haiku
```

Comment:
Can you remove the comment if it's working now? (ideally by amending your existing commit, and or squashing the two commits together, to keep history clean)